federation/develop/develop.sh

```diff
@@ -123,7 +115,9 @@ function get_version() {
     kube_version="${KUBE_VERSION}"
   else
     # Read the version back from the versions file if no version is given.
-    kube_version="$(jq -r '.KUBE_VERSION' ${VERSIONS_FILE})"
+    kube_version="$(cat ${VERSIONS_FILE} | python -c '\
```

Comment:
I am not sure if this is the best way to do this.
Feel free to cc someone else who would know more about this